### ISSUE TITLE
[Bug] Cannot read properties of undefined (reading 'status')

### ISSUE THREAD

**Issue by paulchartres** [benphelps/homepage#1162]
### Description

Whenever I try to load the homepage with several services (containers with docker status), I get an error saying "cannot read properties of undefined (reading 'status')". I tried doing a clean reinstall and adding the services one by one, but I really can't seem to find where it's coming from. I can't see anything noteworthy in the console nor in the logs of the container.
Sometimes it does work after a few refreshes.  
From what I could see, the error happens around here:  
```js
a.ZP)("/api/docker/status/".concat(r.container, "/").concat(r.server || "")), c = o.data;
            return o.error && (c.status,
            i("docker.error")),
```
So I'm assuming it's an issue with the Docker configuration. I don't have a specific config for it, I just mount my /var/run/docker.sock.

### Steps to reproduce

Go on the homepage with several docker services. Sorry I can't make this any more specific, this really is my use case.

### homepage version

latest

### Installation method

Docker

### Browser Logs

````js
framework-c4881a94a5d6a5c8.js:1 TypeError: Cannot read properties of undefined (reading 'status')
    at x (index-c212e5840c40d46c.js:1:7768)
    at Eo (framework-c4881a94a5d6a5c8.js:1:63192)
    at Pu (framework-c4881a94a5d6a5c8.js:1:74880)
    at xi (framework-c4881a94a5d6a5c8.js:1:121315)
    at bs (framework-c4881a94a5d6a5c8.js:1:109368)
    at gs (framework-c4881a94a5d6a5c8.js:1:109296)
    at vs (framework-c4881a94a5d6a5c8.js:1:109159)
    at as (framework-c4881a94a5d6a5c8.js:1:105939)
    at ls (framework-c4881a94a5d6a5c8.js:1:104490)
    at S (framework-c4881a94a5d6a5c8.js:1:138449)
````

### Before submitting, I have made sure to

- [X] Check [the documentation](https://gethomepage.dev/)
- [X] Follow [the troubleshooting guide](https://gethomepage.dev/en/more/troubleshooting/) (please include output above if applicable).
- [X] Search [existing issues](https://github.com/benphelps/homepage/search?q=&type=issues) and [discussions](https://github.com/benphelps/homepage/search?q=&type=discussions).

**Comment by shamoon** [comment]
Can you try `main`?

I think there may have been an issue when your docker server throws an error (at least initially) that was already resolved.

**Comment by paulchartres** [comment]
I made the switch to `main`, but after a few refreshes I'm getting the error again. I also get a different error on iOS devices (`TypeError: undefined is not an object (evaluating 'c.status')`).

**Comment by shamoon** [comment]
Just double-checking you re-created the container etc? Your original error looks precisely like what was fixed in https://github.com/benphelps/homepage/commit/f895a6bf235f6aba49412e794d1c4073edcff4a2 . now https://github.com/benphelps/homepage/blob/main/src/components/services/status.jsx

**Comment by paulchartres** [comment]
I had re-created the container, but I decided to nuke it completely and recreate everything including volumes just to be sure. Seems like everything is working now! Thanks a ton.

**Comment by shamoon** [comment]
Yes, as noted, the original issue is currently fixed in `main`, not `latest`. This issue happens only when the docker server (regardless of what type of connection) has some kind of error. @cyl3x I've resolved that issue in `main` though Im not certain if it was really the same thing or not (it would have also resulted in a frontend error so the same `...undefined reading 'status'` but not necessarily the same cause.

I can't see a reason why `Names` would be empty looking at the docker API https://docs.docker.com/engine/api/v1.37/#tag/Container/operation/ContainerList but thats what your error implies. It might be helpful to figure out why (in your case) as currently now that I've fixed the error, you presumably won't get status / stats for that container(s). Its clearly not a common thing, in general this code has been present for a very long time and other users haven't reported it often.

Feel free to test out `main` with a fresh pull + re-create
